tmt/utils/__init__.py

```
@@ -120,6 +120,24 @@ def configure_constant(default: int, envvar: str) -> int:
         ) from exc
 
 
+def configure_bool_constant(default: bool, envvar: str) -> bool:
```

Just a un-educated thought - since we are deducing bool from `1`,`0` str, why not include `true`/`false`, `yes`/`no` (case insensitively)? It could be useful in other parts of the code, no?

---

Might be silently allowing unexpected input: Click's environment variable ingestion will recognize only `0` and `1` for boolean flags. tmt recognizing more inputs might lead someone to use `export FOO=true` and it would not work with Click processing the options.

---

Ack, thanks for the info!

---

Should we raise an error ? If parsed via click...
```
$ TMT_PLUGIN_EXECUTE_TMT_IGNORE_DURATION=foo tmt_DUR run -a execute -h tmt
Usage: tmt_DUR run execute [OPTIONS]
Try 'tmt_DUR run execute --help' for help.

Error: Invalid value for '--ignore-duration': 'foo' is not a valid boolean.

```